_posts/2020-03-19-quarkus-1-3-0-final-released.adoc

```diff
@@ -0,0 +1,213 @@
+date: 2020-03-19
+tags: release
+synopsis: Quarkus 1.3.0.Final comes with a new class loader infrastructure, GraalVM 20 support, 8 new extensions and a lot of bugfixes and improvements.
+author: gsmet
+---
+
+It has been a bit less than two months since our last major Quarkus release
+as we went through some extensive testing phases to validate our new class loader infrastructure.
+
+A big thanks to all the people who tested our Alphas/CRs and provided feedback.
+
+Two months?
+That's a lot in the Quarkus world so expect a whole lot of new things.
+
+Enough introduction, let's see what 1.3 brings to the world.
+
+== Migration Guide
+
+Eager to migrate your existing applications? Here is https://github.com/quarkusio/quarkus/wiki/Migration-Guide-1.3[our migration guide for 1.3.0.Final].
+
+== What's new?
+
+=== New class loader infrastructure
+
+Due to the innovative nature of Quarkus (especially in our test infrastructure and dev mode), we had a few recurring class loading issues.
+
+Quarkus 1.3 comes with a brand new class loading infrastructure, fixing the structural issues we had.
+The new class loading infrastructure is described at length in the https://quarkus.io/guides/class-loading-reference[Class Loading Reference guide].
+
+We ironed out this new infrastructure during the Alpha and CR phases so it is pretty solid now.
+If you encounter weird class loading issues, report them with a reproducer, we will include all those fixes in the upcoming 1.3.1.
+
+=== One Vert.x to rule them all
+
+A significant amount of work went into having only one Vert.x instance to handle everything in Quarkus instead of two.
+The idea behind this work is to reduce the resource consumption, hopefully without changing anything for you.
+
+=== Banner
+
+This is probably the most visible change:
+by default, an ASCII-art banner is now displayed when starting Quarkus.
+
+You can of course:
+
+ * customize it,
+ * disable it for a given project,
+ * disable it globally for all your projects.
+
+Please refer to https://quarkus.io/guides/getting-started#banner[the documentation] for more details.
+
+=== Mutiny arising
+
+A new reactive programming API has been introduced.
+This API named https://smallrye.io/smallrye-mutiny/[Mutiny] replaces the Axle and Reactive Streams Operators models (Reactive Streams and CompletionStage).
+The previous models are still functional, but deprecated and will be removed in the future.
+
+If you would like to know more, please refer to the https://quarkus.io/guides/getting-started-reactive[Getting Started with Reactive guide].
+
+=== Easy deployment to Kubernetes or Openshift
+
+The Kubernetes extension has been overhauled and now gives users the ability to deploy their Quarkus application to Kubernetes or Openshift with almost no effort.
+Essentially the extension now in addition to generating the Kubernetes manifest files, also takes care of generating a container image (see <<container-image>> for more details)
```

Comment:
Is this the proper way to link to a section withing the doc @gsmet ?

Reply:
Yes, it should work, except there's an `s` missing in the anchor name.

Reply:
Thanks for spotting, fixed!